src/cogs/jaegeraccounts.py

```diff
@@ -161,16 +161,15 @@ async def insert_booking(self, bot: commands.Bot, ctx: commands.Context, url: st
                 last_index = index
                 break
             except ValueError:
-                print(f"`{date}` could not be parsed")
-        # TODO Will need to figure out what to do in case if no dates could be found
+                logging.info(f"`{date}` could not be parsed when when inserting booking")
```

Comment:
I think the no date scenario should be handled with one of the custom exceptions, or maybe a new one

Reply:
Then the errorhandler handles the user feedback

Reply:
We can do that in a later commit if you want tho

Reply:
No date scenario is handled by creating one.
Next couple lines do following:
If last recorded date not found, create a new column and set to today